### DISCUSSION TITLE
mock analysis

### DISCUSSION THREAD

**Discussion by KGadallah** [mboquien/cigale#30] [General]
For 73 AGN galaxies, I had run the X-cigale separately for each galaxy on the laptop. This is because the laptop can't run all in one round.   I collected the results of these separate SEDs and plotted the mock figures (true- vs. estimated- values).  
Has this way been accepted or not?  The correlations in this figure seem too good.  This is due to the improvement of the values of the reduced chi-square. Can anyone help me by running these galaxies in one round on his operating system? If he agrees I will send him the initiate and data files of them.

**Comment by mboquien** [comment]
If your laptop is a little too light, I suggest you set the `blocks` parameter in the `pdf_analysis` module to something larger than 1 (for instance try 10 or 50).

**Reply by KGadallah** [reply]
In fact, my laptop is Intel(R) Core (TM) i7-4510U CPU @ 2.00GHz 2.60 GHz with 16.0 GB RAM. I don't see it is too light. As I put the blocks parameter with different values between 10 and 50, I have partially got an improvement in the compilation but over longer time (i.e. 2 days). For this compilation, the number of the models was almost 34 Millions.  I see that the results still need more values of fit parameters to get the best fit. As soon as I increase these parameters, the compilation can not be done. What I can do? 

To simply estimate the mock file (true- vs. estimated- values as shown in the enclosed file) as shown in the attached file, I would ask if I can compile each source individually to get the best fit? 
![F4](https://user-images.githubusercontent.com/109315794/195983356-be168e49-ba84-48a4-8b19-a945142cd621.png)

**Reply by mboquien** [reply]
From what I can see, this is a low power dual core CPU released over 8 years ago. I am not overly surprised things are a little slow. I am not sure what you mean by compilation but if it still takes too much memory, you can increase the number of blocks. It will be slightly slower but it will go through.
